pkgs/development/libraries/libax25/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, glibc
+, static ? stdenv.hostPlatform.isStatic
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libax25";
+  version = "0.0.12-rc5";
+
+  buildInputs = [ glibc ] ++ lib.optional static [ glibc.static ];
+
+  # Due to recent unsolvable administrative domain problems with linux-ax25.org,
+  # the new domain is linux-ax25.in-berlin.de
+  src = fetchTarball {
+    url = "https://linux-ax25.in-berlin.de/pub/ax25-lib/libax25-${version}.tar.gz";
+    sha256 = "1xnxab6n8w8b00dmg5idb6xwqs3llqsl0wv6s3xkf72pn1p1f7k5";
+  };
```

```suggestion
  src = fetchzip {
    url = "https://linux-ax25.in-berlin.de/pub/ax25-lib/libax25-${version}.tar.gz";
    hash = "sha256-ZR4XbrBXHDf70GZzQDWmdGjMu1ktllcbAAtxZM1S3fY=";
  };
```

You can't use the builtin `fetchTarball` in restricted eval. Please use `fetchzip` instead.

---

good to know! Ill definitely keep that in mind going forward

---

Assuming the tarball is static -- which, in this case, I'd say it is (unless it somehow is dynamically generated, but the URL doesn't make me think it would be) -- I think `fetchurl` would be the better choice here.

---

Updated to `fetchurl`